blazor/datagrid/how-to/enable-or-disable-grid.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Enable or Disable the Blazor Grid | Syncfusion
+description: Learn here all about enable or disable the Syncfusion Blazor DataGrid component and more.
```

Comment:
Learn here all about how to enable or disable the Syncfusion Blazor DataGrid component and more.